modules/adb-lakehouse/variables.tf

```diff
@@ -35,7 +35,7 @@ variable "databricks_workspace_name" {
 
 variable "data_factory_name" {
   type        = string
-  description = "(Required) The name of the Azure Data Factory to deploy"
+  description = "The name of the Azure Data Factory to deploy"
```

Comment:
I think that if we provide it as optional, then we can do it on the module level as well. Imho, ADF should be really optional - it's not a prerequisite for deployment of ADB, as Workflows covers a lot of functionality

Reply:
IMO we can remove ADF from the module. Customers can customise the module to add it if needed.

Reply:
made optional in module in https://github.com/databricks/terraform-databricks-examples/pull/68/commits/d3554826c6f3238768fd535205f8eabb2b63c9d6